xbmc/pvr/PVRThumbLoader.cpp

```diff
@@ -13,7 +13,7 @@
 #include "TextureCache.h"
 #include "TextureCacheJob.h"
 #include "pictures/Picture.h"
-#include "pvr/PVRGUIDirectory.h"
+#include "pvr/filesystem/PVRGUIDirectory.h"
```

Comment:
wrong include order, this has to be moved one line down

Reply:
oversight on my end, sorry.